api/tests/app/test_area_occupancy_rules.py

```diff
@@ -68,15 +73,21 @@ def test_unitary_set_get_delete(self, config_rollback_areas):
         assert res1["occupancy_rules"][0]["max_occupancy"] == 12
         assert res2["occupancy_rules"][0]["max_occupancy"] == 100
 
-    def test_get_not_found(self, config_rollback_areas):
+        # Rollback the config file for area ID: 537
+        config_directory = config_utils.get_area_config_directory(get_config())
+        config_path = os.path.join(config_directory, str(area_id) + ".json")
```

Comment:
Here as well, prefer the helper function

Reply:
Done!